### ISSUE TITLE
[Feature Request] keyboard shortcut for copying film URL to clipboard

### ISSUE THREAD

**Issue by kynan** [mediathekview/MediathekView#622]
Would be great to have a keyboard shortcut for copying the URL of the currently selected title in highest quality to the clipboard.

**Comment by styroll** [comment]
The shortcuts are listed [here](https://mediathekview.de/changelog/13-3-0/), but some may not work anymore depending on the OS you use:

```
Tab “Filme” Key commands (9 Kommandos):

    p oder ENTER = Film abspielen
    d = Film downloaden
    h = HD URL kopieren
    n = Normale URL kopieren
    k = Kleine URL kopieren
    m = MediaDB
    g = Film als gesehen markieren
    u = Film als ungesehen markieren

Tab “Downloads” Key commands (6 Kommandos):

    ENTER = Download bearbeiten
    ENTF = Download löschen
    g = Film als gesehen markieren
    u = Film als ungesehen markieren
    d = markierten Download starten
    m = In MediaDB suchen
```
And as I suggested [here](https://forum.mediathekview.de/post/16413), the shortcuts should be visible in the menus.

**Comment by derreisende77** [comment]
@styroll These are "key" commands relevant *only* to the focused Table view. And menu commands **must** be unique, they can be applied to duplicate menu entries - but the behaviour is unpredictable and erratic. This would require a complete restructure for entirely new menu hierarchy.
@kynan Maybe I can provide a command like Ctrl-Alt-Shift-H for high quality URL to Clipboard. Would that be sufficient?

**Comment by kynan** [comment]
Sure, that would be great, thanks!

**Comment by derreisende77** [comment]
will be available in 14.0.1

**Comment by kynan** [comment]
Is the list of supported keyboard shortcuts available anywhere else but the changelog?
